osu.Server.Queues.ScoreStatisticsProcessor/Processors/MedalProcessor.cs

```diff
@@ -43,12 +43,19 @@ static MedalProcessor()
 
         public bool RunOnLegacyScores => false;
 
+        // ReSharper disable once UselessBinaryOperation
+        // This processor needs to run after the play count and hit statistics have been applied.
+        public int Order => Math.Max(PlayCountProcessor.ORDER, HitStatisticsProcessor.ORDER) + 1;
```

Comment:
Would it not make more sense to just make this always run last? ie. give it a very very high `ORDER`?

Reply:
Something like

```diff
diff --git a/osu.Server.Queues.ScoreStatisticsProcessor/Processors/MedalProcessor.cs b/osu.Server.Queues.ScoreStatisticsProcessor/Processors/MedalProcessor.cs
index 22afa78..c93862e 100644
--- a/osu.Server.Queues.ScoreStatisticsProcessor/Processors/MedalProcessor.cs
+++ b/osu.Server.Queues.ScoreStatisticsProcessor/Processors/MedalProcessor.cs
@@ -45,7 +45,7 @@ namespace osu.Server.Queues.ScoreStatisticsProcessor.Processors
 
         // ReSharper disable once UselessBinaryOperation
         // This processor needs to run after the play count and hit statistics have been applied.
-        public int Order => Math.Max(PlayCountProcessor.ORDER, HitStatisticsProcessor.ORDER) + 1;
+        public int Order => int.MaxValue;
 
         public void RevertFromUserStats(SoloScoreInfo score, UserStats userStats, int previousVersion, MySqlConnection conn, MySqlTransaction transaction)
         {

```

Reply:
Wouldn't hurt I suppose.